books/free-programming-books-es.md

```diff
@@ -235,6 +235,7 @@
 #### Node.js
 
 * [El Libro para Principiantes en Node.js](https://www.nodebeginner.org/index-es.html) -  Manuel Kiessling, Herman A. Junge (HTML)
+* [How To Code in Node.js](https://www.digitalocean.com/community/books/how-to-code-in-node-js-ebook) - Stack Abuse
```

Comment:
not spanish
Please read CONTRIBUTING.md guidelines!

Reply:
> not spanish Please read CONTRIBUTING.md guidelines!

Oh I am sorry I will check and update